README.md

```diff
@@ -23,46 +23,51 @@ python3 -m pip install --extra-index-url https://artifacts.luxonis.com/artifacto
 ### Dependencies
  - cmake >= 3.4
  - C++14 compiler (clang, gcc, msvc, ...)
- - Python
+ - Python3
 
 Along these, dependencies of depthai-core are also required
 See: [depthai-core dependencies](https://github.com/luxonis/depthai-core#dependencies)
 
 
 ### Building
 
-To build a shared library from source perform the following:
+To build and install using pip:
 ```
-git submodule update --init --recursive
-mkdir build && cd build
-cmake .. [-D PYTHON_EXECUTABLE=/full/path/to/python]
-cmake --build . --parallel
+python3 -m pip install .
```

Comment:
Should we recommend, or mention `-v` option, so status can be seen?

Reply:
We could although if it fails, its printed.
Not sure if usable otherwise for someone who just wants to install.

Will briefly describe the parameter below